client/layout/masterbar/logged-in.jsx

```diff
@@ -845,6 +841,7 @@ class MasterbarLoggedIn extends Component {
 				<div className="masterbar__section masterbar__section--right">
 					{ this.renderCart() }
 					{ this.renderLaunchpadNavigator() }
+					{ this.renderReader( ! isMobile ) }
```

Comment:
since we are removing the showLabel argument above, we could remove the `! isMobile` input here as well.

Reply:
Thanks for catching that! Removed.